internal/fptest/ports.go

```diff
@@ -15,11 +15,33 @@
 package fptest
 
 import (
+	"fmt"
 	"sort"
+	"testing"
 
 	"github.com/openconfig/ondatra"
 )
 
+// PortChannelName returns the port channel interface name which is
+// vendor specific.
+func PortChannelName(t testing.TB, dut *ondatra.DUTDevice, i int) string {
```

Comment:
I know that PortChannelName is coming from the internal test that we've already written, but on second thought I'd like to make some changes.

1. I think RT-5.2 is the only test that uses this function, so let's move this to aggregate_test.go and make it unexported.
2. Rename the function to lagName so it's vendor neutral.

Reply:
This is also used by the gnmi_1_5 test.  I considered moving it into the test, but this seems like a helper that is necessary for any test involving aggregate interfaces.  Given that it is used in multiple tests, should we duplicate it in each test?

I agree on point 2.

Reply:
I see. In that case, let's add some unit tests for LAGName(), for both positive and negative cases, and maybe consider rewriting the function to remove the dependency on `t testing.TB` by returning an error instead?